### PR TITLE
enable parallelism and workflow

### PR DESCRIPTION
**Pull request by MalloZup**
# description:

enable backport from the operators

### CODE REVIEW

**Comment by MalloZup** [comment]
why it doens"t run

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+## What does this PR change?
```
Comment (MalloZup) [inline .github/PULL_REQUEST_TEMPLATE.md]: please review this PR_TEMPLATE also

Comment (inercia) [reply .github/PULL_REQUEST_TEMPLATE.md]: I would add a _how to test this by the QA guys (if it applies)_ section...

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+**add description**
+
+## Previous behaviour
```
Comment (Klaven) [inline .github/PULL_REQUEST_TEMPLATE.md]: this section seems redundant to "What does this PR change?"

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+- [ ] **DONE**
+
+## Documentation
+- No documentation needed: **add explanation.**
+- [ ] **DONE**
+
+## Test coverage
+- No tests: **add explanation**
+- Unit tests were added
+- E2E tess were added
```
Comment (Klaven) [inline .github/PULL_REQUEST_TEMPLATE.md]: is it possible for someone to add e3e tests that does not have access to the SUSE CI?

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (MalloZup) [reply .github/PULL_REQUEST_TEMPLATE.md]: yop fixed thx

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 3 above)
Comment (MalloZup) [reply .github/PULL_REQUEST_TEMPLATE.md]: @Klaven  yop this is is the long/term mid-term goal

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (MalloZup) [reply .github/PULL_REQUEST_TEMPLATE.md]: added thx

**Comment by MalloZup** [comment]
wating for green and merging. we can still improve the PR-template later

**Pull request merged**